### ISSUE TITLE
VSIX Vitest Extension not providing UI button or recognizing tests

### ISSUE THREAD

**Issue by mujevich** [vitest-dev/vscode#286]
### Describe the bug

I had initially set up my project using the now deprecated [Vitest Extension](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer), and have now migrated to the [currently supported version ](https://marketplace.visualstudio.com/items?itemName=vitest.explorer)

My issue is that after migrating, my test no longer have the Play / Debug options next to them and the tests don't seem to be recognized. I'm still able to run my tests via the command line however.

![image](https://github.com/vitest-dev/vscode/assets/137452016/a4eba3c5-ce9e-43a5-9e79-69b12941207f)

### Reproduction

N.A

### System Info

```shell
"vitest.enable": true,
"vitest.changeBackgroundColor": true,
"vitest.commandLine": "yarn test",
"vitest.watchOnStartup": true,
"vitest.debugExclude": [
    "<node_internals>/**",
    "**/node_modules/**"
]
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vscode/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vscode/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vscode/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vscode/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by mujevich** [comment]
I'm not sure how to reproduce my VsCode setup, or what I'm missing

**Comment by StarpTech** [comment]
I have the same issue. Can be reproduced with https://github.com/wundergraph/cosmo Open it from the root and navigate to https://github.com/wundergraph/cosmo/blob/main/cdn-server/cdn/test/cdn.test.ts

**Comment by mujevich** [comment]
Thanks a ton for that @StarpTech !!

**Comment by ffMathy** [comment]
I have a strong feeling this will become way better after #253 is merged.

**Comment by mujevich** [comment]
Also potentially related to [#285](https://github.com/vitest-dev/vscode/issues/285), when I restarted my app I did see a similar error

![image](https://github.com/vitest-dev/vscode/assets/137452016/723a33bb-3e8e-4db0-a205-45c0fb77b1de)

**Comment by sheremet-va** [comment]
This should work in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.

If you still have the issue, please open a new issue with a reproduction.

**Comment by StarpTech** [comment]
pre-release 0.5.0 is not available on the marketplace.

**Comment by sheremet-va** [comment]
> pre-release 0.5.0 is not available on the marketplace.

https://marketplace.visualstudio.com/items?itemName=vitest.explorer

<img width="283" alt="Screenshot 2024-03-15 at 20 40 49" src="https://github.com/vitest-dev/vscode/assets/16173870/47b64b2c-7a78-44c6-8fdb-3f4d315d15a2">

**Comment by StarpTech** [comment]
Hi, I tested pre-release 0.5.0 and it is still not functional. I also upgraded to vitest 1.4.0 and reload vscode

![image](https://github.com/vitest-dev/vscode/assets/1764424/0abe2136-f5ab-4dce-a86b-a15d39eccf78)

**Comment by sheremet-va** [comment]
In your case, the problem is the monorepo structure. You define workspace configurations for your pnpm workspace, but you don't do the same for Vitest workspace - I would recommend adding a `vitest.workspace` file to improve performance of Vitest, at least: https://vitest.dev/guide/workspace.html

There are other issues at play that I will push the fix, but I would strongly recommend adding a workspace configuration.

**Comment by StarpTech** [comment]
It worked. I also had to install vitest in the root. Thank you.
